proposals/stac_modify.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id": "stac_modify",
+    "summary": "Updates an existing STAC resource",
+    "description": "Modifies the given STAC resource (e.g., a STAC Collection or Item) based on the given changeset.",
```

Comment:
I'm trying to figure out if I would need to use this process or `export_collection` in the scenario where a user created already a STAC Collection with some Items offered via an API and wants to add some new Items.

Reply:
No, this is more to update metadata in a specific resource, not to add items or so.